plans/COHORT_CURATION_PLAN.md

```
@@ -220,6 +220,15 @@ analyses never run per-PR.
 
 ## 5. Order of work
 
+*Status (2026-07-13): step 1 is essentially done. Review batch 5
+(PR #260) fills the last empty class, `ring_only_flat` (five rank-1
+frames plus a sixth 2-D frame, N1467344214, routed to `ring_only_curved`
+with the Phase D reconciliation because the DT gate over-rejects it,
+#261). The Phase D frame reconciliation (branch `phase-d-reconciliation`)
```

markdownlint-cli2 (0.23.0)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)